Dockerfile.test

```diff
@@ -1,27 +1,33 @@
 FROM ubuntu:18.04
-
 ENV INSTALL_DIR=/opt/conjur-api-python3
-
 RUN apt-get update && \
-    apt-get install -y bash \
-                       binutils \
-                       build-essential \
-                       curl \
-                       git \
-                       jq \
-                       libffi-dev \
-                       libssl-dev \
-                       python3 \
-                       python3-dev \
-                       python3-pip
-
+  apt-get install -y bash \
+            binutils \
+            build-essential \
+            curl \
+            git \
+            jq \
+            libffi-dev \
+            libssl-dev \
+            python3 \
+            python3-dev \
+            python3-pip \
+            gnome-keyring \
+            dbus-x11
 RUN mkdir -p $INSTALL_DIR
 WORKDIR $INSTALL_DIR
 
 # Workaround for https://bugs.launchpad.net/usd-importer/+bug/1794041
 RUN pip3 install --upgrade keyrings.alt
 
-COPY requirements.txt $INSTALL_DIR/
+# Add Kering dbus support
+RUN pip3 install keyring_jeepney
```

Comment:
do we still need jeepney?

Reply:
important to know because we will eventually need to write up prereqs for docs for our users. I am doubtful that we need this